cli/rotate.go

```diff
@@ -142,20 +150,26 @@ func RotateCommand(app *kingpin.Application, input RotateCommandInput) {
 		return
 	}
 
-	newVal, err := newSessionCreds.Get()
-	if err != nil {
-		app.Fatalf(awsConfig.FormatCredentialError(err, input.Profile))
-		return
-	}
+	log.Printf("Waiting for new IAM credentials to propagate")
+	time.Sleep(time.Second * 8)
```

Comment:
8 seconds seems a while, should we just start polling immediately?

Reply:
It seems pretty consistently to take about 8 seconds :)

Reply:
We could suppress errors and poll regularly though.